mlflow/system_metrics/metrics/disk_monitor.py

```diff
@@ -0,0 +1,24 @@
+"""Class for monitoring disk stats."""
+
+import os
+
+import psutil
+
+from mlflow.system_metrics.metrics.base_metrics_monitor import BaseMetricsMonitor
+
+
+class DiskMonitor(BaseMetricsMonitor):
+    """Class for monitoring disk stats."""
+
+    def __init__(self, name="disk"):
+        super().__init__(name)
+
+    def collect_metrics(self):
+        # Get disk usage metrics.
+        disk_usage = psutil.disk_usage(os.sep)._asdict()
+        for k, v in disk_usage.items():
+            if "percent" in k:
+                self._metrics[f"disk_memory_{k}"].append(v)
```

Comment:
Can we call it "disk_usage" instead of disk memory? memory implies random access ephemeral state. We're monitoring actual permanent storage utilization here.

Reply:
sure, done!